### PR TITLE
Bug fixes 1

### PR DESCRIPTION
**Pull request by anpolimus**
- [ ] Virtual Y Premium role should be checked by default as well
- [ ] Fixed operations column at Virtual Y logs 
- [ ] Updated readmes

### CODE REVIEW

File: modules/openy_gc_log/config/install/views.view.virtual_y_logs.yml [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 status: true
 dependencies:
+  config:
+    - user.role.administrator
+    - user.role.virtual_ymca_editor
   module:
     - openy_gc_log
+    - user
+_core:
+  default_config_hash: IznglBTuuCmBnXCbRrnLdgGq2NOzuMQ3axmujUAS5kM
```
Comment (froboy) [inline modules/openy_gc_log/config/install/views.view.virtual_y_logs.yml:11]: Shouldn't this hash be removed?

Comment (anpolimus) [reply modules/openy_gc_log/config/install/views.view.virtual_y_logs.yml:11]: yes, sure. thank you.

**Comment by anpolimus** [comment]
retest

**Review by NightHunterSV — APPROVED** [review]

**Comment by anpolimus** [comment]
retest

**Comment by AndreyMaximov** [comment]
retest

**Review by AndreyMaximov — APPROVED** [review]

**Pull request merged**